In [1]:
import pandas as pd

In [2]:
data_1 = pd.read_excel('DATA.xlsx', sheet_name=0)

/tmp/ipykernel_57525/3088015952.py:1: FutureWarning: Inferring datetime64[ns] from data containing strings is deprecated and will be removed in a future version. To retain the old behavior explicitly pass Series(data, dtype=datetime64[ns])
  data_1 = pd.read_excel('DATA.xlsx', sheet_name=0)


In [3]:
data_street_en = data_1[(data_1['Формат магазина'] == 'Street')]
data_street_ru = data_1[(data_1['Формат магазина'] == 'Стрит')]

In [4]:
data_street_en = data_street_en[(data_street_en['Выборка'] == 'Тестовая')].index
data_street_ru = data_street_ru[(data_street_ru['Выборка'] == 'Тестовая')].index

In [5]:
count_street = data_street_en.shape[0] + data_street_ru.shape[0]

In [6]:
count_street

6

In [7]:
data_2 = pd.read_excel('DATA.xlsx', sheet_name=1)

In [8]:
df = pd.merge(data_1, data_2) 

In [9]:
cols = ['2016-01-01', '2016-02-01', '2016-03-01', '2016-04-01', '2016-05-01', '2016-06-01', 
        '2016-07-01', '2016-08-01', '2016-09-01', '2016-10-01', '2016-11-01', '2016-12-01']
df_ = df[cols]

In [10]:
# df[cols] = df[cols].astype('str')
mean_ = df[(df['Формат магазина'] == 'Мини ТЦ')]

In [11]:
df_mean = mean_[cols]

In [12]:
df_mean = df_mean.astype('float64')

In [13]:
val_2 =df_mean.mean().sum()/12

In [14]:
free_parking_magazines = data_1[(data_1['Парковка'] == 'бесплатная парковка')]

In [15]:
count_free = free_parking_magazines.shape[0]

In [16]:
print(count_street)
print(val_2)
print(count_free)

6
336899.59857713565
156


# Ответы

Таким образом. После небольших вычислений можно дать ответы на поставленные задачи:
1)Сколько магазинов формата стрит в тестовой выборке?
2)Какова средняя выручка магазинов формата мини ТЦ за 2016 год?
3)Сколько магазинов с бесплатной парковкой?


Ответы получились следующие. Среди всех магазинов, формата стрит в тестовой выборке 6 шт.
Средняя выручка магазинов формата мини ТЦ за 2016 год 336899.59857713565. Число не округлено, т.к. не известна валюта в которой представленны числа и размерность.
Магазинов с бесплатной парковкой всего 156 шт.